OpenRA.Game/Game.cs

```diff
@@ -279,7 +279,7 @@ internal static void Initialize(Arguments args)
 				}
 			}
 
-			Sound = new Sound(Settings.Server.Dedicated ? "Null" : Settings.Sound.Engine);
+			Sound = new Sound(Settings.Sound.Engine);
```

Comment:
Do we even need `OpenRA.Platforms.Null` anymore for anything after this, or can we finally get rid of it?